src/Features/LanguageServer/Protocol/Handler/Definitions/GoToDefinitionHandlerBase.cs

```diff
@@ -25,10 +31,9 @@ internal static class GoToDefinitionHandler
                 return locations.ToArrayAndFree();
             }
 
-            var position = await document.GetPositionFromLinePosition(ProtocolConversions.PositionToLinePosition(request.Position), cancellationToken).ConfigureAwait(false);
+            var position = await document.GetPositionFromLinePositionAsync(ProtocolConversions.PositionToLinePosition(request.Position), cancellationToken).ConfigureAwait(false);
 
             var definitionService = document.Project.LanguageServices.GetService<IGoToDefinitionService>();
-
             var definitions = await definitionService.FindDefinitionsAsync(document, position, cancellationToken).ConfigureAwait(false);
             if (definitions != null && definitions.Count() > 0)
```

Comment:
there is symbol navigation service + we need to discuss with LiveShare team how we are supposed to deal with cross language references. work item?

Reply:
https://github.com/dotnet/roslyn/projects/45#card-20873103